setup.cfg

```diff
@@ -131,3 +131,5 @@ exclude_lines =
     coverage: ignore
     ImportError
     ModuleNotFoundError
+    pragma: no cover
+    raise NotImplementedError
```

Comment:
I've been more inclined to have `raise NotImplementedError` show up as not covered by tests in coverage reports as a reminder that something is not implemented that needs to be.  With that said, we don't need the line in an abstract method to show up as a untested line.  Hm...

Reply:
As per Python's documentation, I'll replace `raise NotImplementedError` with `...` in the abstract methods.  I'll also remove the additions to `exclude_lines` in the coverage settings.